libfwupdplugin/fu-cabinet.c

```diff
@@ -519,6 +634,22 @@ fu_cabinet_parse (FuCabinet *self,
 	if (!fu_cabinet_decompress (self, data, error))
 		return FALSE;
 
+	/* load the firmware.jcat if included */
```

Comment:
Is it always going to be `firmware.jcat`?  Or should it be better to refer to this from one of the XML files and then load that?

Reply:
> from one of the XML files

Which one? It all gets a bit circular if you allow the XML to define the signature filename, and the signature filename verifies the XML... Perhaps we should just load `*.jcat`? I know some vendors rename their files to `firmware.*` but the majority don't.

Reply:
*.jcat sounds like a good idea.

Reply:
Fixed.